test/e2e/lib/util.js

```diff
@@ -20,9 +23,32 @@ const assertValidWhen = (actual, expected, name) => {
 	const ts = +new Date(actual)
 	a.ok(!Number.isNaN(ts), name + ' is not parsable by Date')
 	// the timestamps might be from long-distance trains
-	a.ok(isRoughlyEqual(day + 6 * hour, +expected, ts), name + ' is out of range')
+	const delta = day + 6 * hour
+	if (!isRoughlyEqual(delta, +expected, ts)) {
+		throw new AssertionError({
+			message: name + ' is out of range',
+			actual: ts,
+			expected: `${expected - delta} - ${+expected + delta}`,
+			operator: 'isRoughlyEqual',
+		})
+	}
 }
 
+// HTTP request mocking
+if (process.env.VCR_MODE && !process.env.VCR_OFF) {
+	const replayer = require('replayer')
+	replayer.configure({
+		headerWhitelist: [
+			'Content-Type', 'Accept-Encoding', 'Accept',
+		],
+		includeHeaderValues: true,
+		touchHits: false,
+	})
+	replayer.fixtureDir(join(__dirname, '..', 'fixtures'))
+}
+const test = tape
```

Comment:
Shouldn't we still use `tape-promise` here, since requests are technically still async and one could disable `process.env.VCR_MODE`?

Reply:
[`tape@5` supports async test functions.](https://github.com/substack/tape/blob/3554d4b049f34065c6a643c5cda3d88c70a4908e/lib/test.js).

Reply:
ah, nevermind then.